events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
 - Work alongside the CNCF Events Team to coordinate communications efforts
 - Assist with deck content for curated talks and/or other purposes
+- Onsite, a Comms lead acts as the contact person for any comms needed onsite, such as messaging about room or registration changes that may occur onsite.
+
+### What to do as KCS Comms Lead
+This section acts as a checklist for activities the KCS Comms Lead is likely to do each cycle.
+
+- Create Slack DM with Comms team (lead, shadows, volunteers)
+- Create a doc using the [KCS Comms Team Plan Template] & put it in the relevant KCS folder
+  - Update shadow & volunteer requirements to suit your team
+  - Update Staff Summit Meeting sign up sheet
+- Create Email Plan Doc & put it in the relevant KCS folder
+- [Optional] Set up meetings with Comms team
+- Claim and write the announcement comms (email &/or social, slack, etc.)
+- Share Plan Docs with team
+  - Ensure shadows and volunteers are signed up for necessary activities
+- Ensure comms leading up to the event are planned, prioritized, and completed successfully
+- [Post-KCS] Update role handbook
+- [Post-KCS] Update KCS Comms Overall Plan Template doc
+
+## Shadow to the KCS Comms Lead Role
+
+Shadows are expected to serve as KCS Comms Leads in a future cycle. During their shadowing cycle(s), Shadows will work closely with the KCS Comms Lead to learn both how to run the KCS Comms team and how to create comms themselves. By signing up to shadow, you are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). A Shadow candidate must either have [Kubernetes Org Membership] or be planning to pursue org membership.
+
+[Kubernetes Org Membership] can be obtained through KCS Comms work, and a shadow should complete enough tasks to be eligible for org membership over the course of a cycle. Do note that org membership is an indication of active and continuous contribution to the Kubernetes project/community. Shadows are expected to be committed to contributing to Kubernetes on a long-term basis.
+
+### Requirements
+* Review this document.
+* A KCS Comms Shadow must either have, or plan to pursue, [Kubernetes Org Membership].
+* Commit to contributing to the KCS Staff team for multiple cycles.
+
+## Tasks
+During the course of their KCS cycle as a shadow, KCS Comms Shadows are expected to complete tasks including but not limited to:
+* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
+* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
+* Representing the KCS Comms team in KCS Staff meetings.
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* Reviewing comms written by other shadows, volunteers, or the lead as directed by the KCS Comms Lead
```

Comment:
I would drop the, "as directed..." part here as I think it's a must for even the leads' work to be reviewed.